### PR TITLE
enable building the hypershift images for arm64 architecture

### PR DESCRIPTION
**Pull request by droslean**
/cc @aleskandro @danilo-gemoli @openshift/test-platform

### CODE REVIEW

File: ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.13-postsubmits.yaml [hunk 1]
```diff
@@ -1,5 +1,60 @@
...
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --image-mirror-push-secret=/etc/push-secret/.dockerconfigjson
+        - --promote
+        - --report-credentials-file=/etc/report/credentials
+        - --target=[images]
+        - --target=test-bin
+        - --variant=arm64
+        command:
+        - ci-operator
+        image: ci-operator:latest
```
Comment (droslean) [inline ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.13-postsubmits.yaml:28]: I missed the postsubmits: https://github.com/openshift/ci-tools/pull/3614

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main__arm64.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  image_stream_tag:
+    name: builder
+    namespace: ocp-arm64
+    tag: rhel-8-golang-1.19-openshift-4.14
+images:
+- from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift-arm64/release:golang-1.19
```
Comment (aleskandro) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main__arm64.yaml:25]:
```suggestion
      - registry.ci.openshift.org/openshift/release:golang-1.19
```

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main__arm64.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  to: hypershift-operator
+- dockerfile_path: Dockerfile.control-plane
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.19-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp-arm64/builder:rhel-8-golang-1.19-openshift-4.14
```
Comment (aleskandro) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main__arm64.yaml:32]:
```suggestion
      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14
```

**Comment by droslean** [comment]
https://github.com/openshift/release/pull/43246 for the `generated-config` job

**Comment by bryan-cox** [comment]
@droslean - We need this fro 4.13 and 4.14 as well. Should I open a PR for those?
cc: @aleskandro

**Comment by droslean** [comment]
I can open the PR, but I will need your guidance on the images we need to use. There is limited availability for arm64 though. Also, the postsubmits are failing now because we don't fully support them. We are working on automating this as well. In the meantime, we can do a call to discuss this further.

**Comment by bryan-cox** [comment]
Ok, sounds good on opening the PR.

**Pull request merged**